src/hotspot/share/services/diagnosticCommand.cpp

```diff
@@ -554,13 +554,29 @@ int HeapDumpDCmd::num_arguments() {
 ClassHistogramDCmd::ClassHistogramDCmd(outputStream* output, bool heap) :
                                        DCmdWithParser(output, heap),
   _all("-all", "Inspect all objects, including unreachable objects",
-       "BOOLEAN", false, "false") {
+       "BOOLEAN", false, "false"),
+  _parallel_thread_num("-parallel",
+       "Number of parallel threads for heap iteration. "
+       "0 means let the VM determined the number of threads. "
```

Comment:
Should be "determine". Sorry that was a typo in the text I suggested.

Reply:
My bad, thanks for pointing out. :-)